<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Align anchor point</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>16</number>
   </property>
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>16</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QToolButton" name="topLeft">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/ap-top-left.svg</normaloff>icons/ap-top-left.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">anchorPoint</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="topCenter">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/ap-top-center.svg</normaloff>icons/ap-top-center.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">anchorPoint</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="topRight">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/ap-top-right.svg</normaloff>icons/ap-top-right.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">anchorPoint</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="centerLeft">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/ap-center-left.svg</normaloff>icons/ap-center-left.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">anchorPoint</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="centerCenter">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/ap-center-center.svg</normaloff>icons/ap-center-center.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">anchorPoint</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="centerRight">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/ap-center-right.svg</normaloff>icons/ap-center-right.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">anchorPoint</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QToolButton" name="bottomLeft">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/ap-bottom-left.svg</normaloff>icons/ap-bottom-left.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">anchorPoint</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="bottomCenter">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/ap-bottom-center.svg</normaloff>icons/ap-bottom-center.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">anchorPoint</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="bottomRight">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/ap-bottom-right.svg</normaloff>icons/ap-bottom-right.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">anchorPoint</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="anchorPoint"/>
 </buttongroups>
</ui>
